src/aerie_cli/persistent.py

```diff
@@ -21,15 +21,53 @@
     APP_DIRS.user_config_dir).resolve().absolute()
 CONFIGURATION_FILE_PATH = CONFIGURATION_FILE_DIRECTORY.joinpath('config.json')
 
-SESSION_FILE_DIRECTORY = Path(APP_DIRS.user_config_dir).resolve().absolute()
+SESSION_FILE_DIRECTORY = Path(APP_DIRS.user_config_dir).resolve().absolute() / "session_files"
+SESSION_FILE_DIRECTORY.mkdir(parents=True, exist_ok=True)
 SESSION_TIMESTAMP_FSTRING = r'%Y-%jT%H-%M-%S.%f'
 SESSION_TIMEOUT = timedelta(hours=12)
 
+MAX_LOG_FILES = 25
+
+TIME_FORMAT = '%Y-%m-%d_%H:%M:%S'
+START_TIME = datetime.now().strftime(TIME_FORMAT)
+LOGS_PATH = Path(APP_DIRS.user_config_dir).resolve().absolute() / "logs"
+LOGS_PATH.mkdir(parents=True, exist_ok=True)
+CURRENT_LOG_FILE_NAME = f"aerie_cli_{START_TIME}"
+CURRENT_LOG_PATH = LOGS_PATH / (f"{CURRENT_LOG_FILE_NAME}.log")
+number_appended_to_log = 1
+while CURRENT_LOG_PATH.exists():
+    CURRENT_LOG_PATH = LOGS_PATH / (f"{CURRENT_LOG_FILE_NAME}_{number_appended_to_log}.log")
 
 def delete_all_persistent_files():
     shutil.rmtree(CONFIGURATION_FILE_DIRECTORY, ignore_errors=True)
     shutil.rmtree(SESSION_FILE_DIRECTORY, ignore_errors=True)
-
+    shutil.rmtree(LOGS_PATH, ignore_errors=True)
+    # The configuration file directory is the parent of the session and log directories.
+    # Those are only initialized at the start of the program,
+    # so we must re-create them here to prevent a 'directory doesn't exist' error
+    SESSION_FILE_DIRECTORY.mkdir(parents=True, exist_ok=True)
+    LOGS_PATH.mkdir(parents=True, exist_ok=True)
+
+def clear_old_log_files():
```

Comment:
Same comment as above -- this probably ought to be a `PersistentLogs.clear_old_log_files` method